uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/LeaveRequestTest.php

```diff
@@ -696,4 +703,449 @@ public function testCalculateBalanceChangeWithAllRequiredParameters() {
     ]);
     $this->assertEquals($expectedResultsBreakdown, $result['values']);
   }
+
+  public function testLeaveRequestIsValidWithoutAStartDate() {
+    $result = civicrm_api3('LeaveRequest', 'isvalid', [
```

Comment:
Overall comment about the tests on this class: The test names are confusing. They're saying the opposite of what is a valid Leave Request. Let's take this one here as an example. If you remove the "test" prefix, it says: "Leave Request Is Valid Without A Start Date", which is wrong. Note that the "test" prefix is only that so phpunit knows it's a test and it's not part of the test name.

Take a look of how the tests will look like when you run phpunit4 with the --testdox option:
![screen shot 2016-12-21 at 12 29 31](https://cloud.githubusercontent.com/assets/388373/21392941/5a875c48-c779-11e6-8dbc-f03fb1827f2a.png)

A suggestion for a better names would be something like "testLeaveRequestIsValidShouldReturnErrorWhenStartDateIsEmpty"